src/ring_theory/power_series/basic.lean

```diff
@@ -679,9 +679,6 @@ end⟩
 
 variables [local_ring R] [local_ring S]
 
```

Comment:
These lines now also do nothing useful, right?
```suggestion
```

Reply:
Oh good catch! Let's check...

Reply:
Unsurprisingly, you are right :-)